test/e2e/test_cluster_configs/everything.json

```diff
@@ -14,6 +14,8 @@
 		"properties": {
 			"orchestratorProfile": {
 				"kubernetesConfig": {
+					"networkPlugin": "azure",
```

Comment:
These changes ensure we still have test coverage for Azure CNI + bridge mode now that Azure CNI 1.2.0 has merged with transparent mode enabled by default.